helm-charts/stable/cloudwatch-agent/templates/cloudwatch-agent-service-account.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{ if .Values.cloudwatchAgent.serviceAccount.create -}}
```

Comment:
Does this service account need AWS iam role access? If not, then we can just create the service account through the helm chart instead of through the CDK.

Reply:
Yes, it requires, iam permissions.